test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_to_VR_DelCmd_WARNINGS_to_UI_DelCmd.lua

```diff
@@ -0,0 +1,167 @@
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+function Test.Precondition_PutFile()
+  commonSteps:PutFile("Precondition_PutFile", "icon.png")
+end
+
+function Test:Precondition_AddCommand()
+  local cor_id_add_cmd = self.mobileSession:SendRPC("AddCommand",
+  {
+    cmdID = 11,
+    menuParams =  
+    { 
+      parentID = 0,
+      position = 0,
+      menuName ="Commandpositive"
+    }, 
+    vrCommands = 
+    { 
+      "VRCommandonepositive",
+      "VRCommandonepositivedouble"
+    }, 
+    cmdIcon =   
+    { 
+      value ="icon.png",
+      imageType ="DYNAMIC"
+    }
+  })
+
+  EXPECT_HMICALL("UI.AddCommand", 
+  { 
+    cmdID = 11,
+    cmdIcon = 
+    {
+      value = storagePath.."icon.png",
+      imageType = "DYNAMIC"
+    },
+    menuParams = 
+    { 
+      parentID = 0, 
+      position = 0,
+      menuName ="Commandpositive"
+    }
+  })
+
+  :ValidIf(function(_,data)
+    local value_Icon = storagePath .. "action.png"
+    if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."action.png)" .. "$") == nil ) then
+      print("\27[31m value of menuIcon is WRONG. Expected: ~".. value_Icon .. "; Real: " .. data.params.cmdIcon.value .. "\27[0m")
+    return false
+    else
+    return true
+    end
+  end)
+
+  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+
+  EXPECT_HMICALL("VR.AddCommand", 
+  { 
+    cmdID = 11,
+    type = "Command",
+    vrCommands = { "VRCommandonepositive", "VRCommandonepositivedouble" }
+  })
+  :Do(function(_,data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  
+  EXPECT_RESPONSE(cor_id_add_cmd, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange")
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_DeleteCommand()
+  local cor_id_del_cmd = self.mobileSession:SendRPC("DeleteCommand",
+  {
+    cmdID = 11
+  })  
+
+  EXPECT_HMICALL("UI.DeleteCommand", 
+  { 
+    cmdID = 11,
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :Do(function(_,data)
+    self.hmiConnection:SendResponse(data.id, "UI.DeleteCommand", "WARNINGS", {})
+  end)  
+
+  EXPECT_HMICALL("VR.DeleteCommand", 
+  { 
+    cmdID = 11,
+    type = "Command",
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :Do(function(_,data)
+    self.hmiConnection:SendResponse(data.id, "VR.DeleteCommand", "UNSUPPORTED_RESOURCE", {})
```

Comment:
@istoimenova , script was updated in commit  d2baa99d69379db10009b64a0949b062a9e6933f